packaging/linux/systemd/keybase.gui.service

```
@@ -19,7 +19,7 @@ EnvironmentFile=-%t/keybase/keybase.gui.env
 EnvironmentFile=-%h/.config/keybase/keybase.autogen.env
 EnvironmentFile=-%h/.config/keybase/keybase.env
 
-ExecStart=/opt/keybase/Keybase
+ExecStart=/usr/bin/env bash -c "/opt/keybase/Keybase $KEYBASE_STARTUP_URL"
```

```suggestion
ExecStart=/bin/sh -c "/opt/keybase/Keybase $KEYBASE_STARTUP_URL"
```

This does not use any bashisms, /bin/sh is guaranteed to exist on all Unix-like systems and may be more lightweight than bash.

Can KEYBASE_STARTUP_URL contain spaces?